src/integrations/prefect-dbt/prefect_dbt/cli/commands.py

```diff
@@ -195,8 +195,7 @@ def _stream_output(event):
         run_results = consolidate_run_results(result)
         markdown = create_summary_markdown(run_results, command)
         artifact_id = await create_markdown_artifact(
-            markdown=markdown,
-            key=summary_artifact_key,
+            markdown=markdown, key=summary_artifact_key, _sync=False
```

Comment:
key change

Reply:
`_sync=False` feels like syntax we're _eventually_ going to move away from - wouldn't it be better to call `acreate_markdown_artifact` here?

Reply:
yes it would, updated in [d4662fb](https://github.com/PrefectHQ/prefect/pull/16926/commits/d4662fbc95735ed8c212b3b1802cd1756591549c)